### ISSUE TITLE
Request to get "CurrentTemperature" values hijacked by Weather skill

### ISSUE THREAD

**Issue by FloatingBoater** [openhab/openhab-mycroft#79]
Hi,

After testing "Current Temperature" tagging as part of a skill discussion over on the openHAB forum, it looks like the Weather skill is high-jacking all requests to ask for a temperature sensor value:
https://community.openhab.org/t/mycroft-openhab-skill/33475/45?u=floatingboater

# Platform
Mycroft Mark 1 hardware
mycroft-core v19.8.4
openHAB 2.5.0.M5 Milestone Build

# openHAB config
Sample `*.items` definitions:
```
Switch  sBed1_Lamps         "Bedside Light"             <light> (gU_Bed1, gLights)  ["Lighting"]    {channel="zwave:device:aaaaaaaa:node99:switch_binary"}
Number  owRoom1          "Inside  [%.1f C]"  <temperature>   ["CurrentTemperature"]  {channel="onewire:temperature:aaaaaaaa:99_000000000000:temperature"}
```

# Behaviour
SSH’d directly into a Mark_1, used mycroft-cli-client to inject speech commands:
```
refresh openhab items
 >> All right. I found 21 items.

list openhab items
 >> Found the following items:
  Lights: Bedside Light <and others> 
  Switches: Good Night <and others>
  Current Temperature: Inside, Radiator <and others>
```
This suggests the Lights, Switches, and Temp items are all loaded into the skill from openHAB.

Example dialogue:
```
Hey, Mycroft
Turn on bedside light  - WORKS AOK
Switch on on bedside light  - WORKS AOK

What is the temperature Inside?
What is the Inside temperature?
What is Inside temperature?
What's Inside temperature?
```
All of the temperature questions just get a generic weather forecast, suggesting the Padatious parser is choosing the Weather skill over the openHAB one to answer.
Other sensors have the same result - e.g. 'Radiator`:
```
get radiator status
 >> I don't know what that means.
 what is radiator temp?
 >> I'm not sure how to help you with that.
 what is radiator temperature?
 >> It's currently 4 degrees celsius.       <This is a weather forecast!>
```

Sadly, I don't see anything useful in `/var/log/mycroft/skills.log`.

After reading `vocab/en-us/WhatStatusKeyword.voc`,  I wonder if an update to the vocab to be more specific about the speech dialogue is needed.
Instead of `what is`, perhaps `what is the temperature {}`?
I'm also trying to go my head around the changes to intent parser - wondering if a port to the latest version might help, especially with translations.

**Comment by mortommy** [comment]
Hi. I was following the discussion. I don't know if there's a way to make mycroft core to choose the correct skill between openHAB and weather. I'll make some tests and look for help among mycroft developers. First of all, I have to set up the v19 environment.

**Comment by FloatingBoater** [comment]
Yes - I absolutely understand this is likely to be rather difficult to resolve, as the keyword temperature is getting caught by what is a core skill for any voice assistant. 

I really appreciate the effort you and other contributors have gone to to get any skill working on Mycroft, especially as the platform and APIs develop and need regular maintenance.

After other threads with openHAB developers stating they prefer tracking issues in GitHub rather than the forum, this report was more for documentation and information. 

My first thought would be to change the language to give more differentiation between 'what is the temperature in x' (meaning weather) and 'what is the temperature of x' (meaning sensor).
Some devices use prefixes (Ask openHAB, what is the living room temperature), but that feels a little awkward.

I remember reading the blog post from (I think) Steve Penrod when the latest parser API was announced - it suggested more of a parallel bus system, with several skills being able to be triggered by one command based on certainty. Sadly, I (yet!) haven't managed to understand how the trigger phrases are defined, especially with the API changes to add support for many languages.

I'm also over on the Mycroft forum with the same username, so will try to help as much as possible.

Thanks!

**Comment by mortommy** [comment]
fixed by #85

**Comment by mortommy** [comment]
fixed by #85

**Comment by danbz** [comment]
Getting exactly this behaviour repeating 

Hardware:PI$ 4GB
OS: Openhabian Install 2.5.10-1
//

Hardware:PI$ 4GB
Picroft Buster Keaton - Pork Pi mycroft-core: 20.8.0 ( master )

checking openhab items via SSH shows
```
Current Temperature: Middle floor, Ground floor, Top floor    
```
and in response to
```
 What's the temperature of Middle floor   
 >> It's currently 8 degrees celsius.   

 whats top floor temperature 
 >> It's currently 8 degrees celsius.   

tell me the temperature of Top floor    
 >> It's currently 8 degrees celsius.         
```

gives the external weather temperature
